src/StoreKeeper/WooCommerce/B2C/Endpoints/FileExport/ExportEndpoint.php

```diff
@@ -30,6 +32,13 @@ public function handle()
 
         $exportClass = FileExportTypeHelper::getClass($type);
         $export = new $exportClass();
+
+        if (ProductFileExport::class === $exportClass || AllFileExport::class === $exportClass) {
```

Comment:
```suggestion
        if ($export instanceof ProductExportInterface) {
```
Better to check for interface.

Reply:
Added `ProductExportInterface`.